functions/__tests__/functions.update.undefined.fields.test.ts

```diff
@@ -94,6 +94,7 @@ describe('extension', () => {
       const payload = {
         ...testDocument,
         'objectID': afterSnapshot.id,
+        'path': afterSnapshot.ref.path,
```

Comment:
this means every existing index will receive path as well right? what do you think about passing the whole snapshot to transform instead (so users can pick which items they need) so it doesn't change the default output